src/app/main/main.tpl.html

```diff
@@ -1,4 +1,4 @@
-<cru-nav></cru-nav>
+<cru-nav language="United States / English"></cru-nav>
```

Comment:
Not sure where the list comes from, but it would seem better to use a code here instead of the translated (or maybe always english) text. Ex: en-US

Reply:
Then we would need a database of lang codes on the JS side.  I'll search around, maybe there is a library.

Reply:
Maybe something like this code work.  https://github.com/bdswiss/country-language

@OzzieOrca What are you thoughts?

Reply:
Something like that could work. I know for the EP stuff we're doing API requests to get countries but that probably doesn't help for languages. It kinda looks like you are getting languages from AEM https://github.com/CruGlobal/give-web/pull/703/files#diff-6d445de8f81bf08615bab285832fae9aR20. I'm not sure I fully understand what you are trying to do.

Reply:
![screen shot 2017-11-07 at 2 35 54 pm](https://user-images.githubusercontent.com/4603906/32513721-fd914c36-c3c8-11e7-8e2f-bdc658c3e44c.png)

The language-picker-lable is used for the link to expand the picker above.  The list in the picker does come from AEM, just like the global overlay did.

Reply:
Ok. What would the language library be for then? Just prefilling the US/English text? Or for translating some of this text?

Reply:
That attribute will be set by AEM, so I think the text label is fine.

Reply:
k